src/containerbuddy/config.go

```diff
@@ -91,12 +91,12 @@ func loadConfig() *Config {
 		backend.onChangeArgs = strings.Split(backend.OnChangeExec, " ")
 	}
 
-	hostname, _ := os.Hostname()
```

Comment:
The hostname defaults to the Docker container ID, which is important in being able to interact with the container elsewhere. Removing it here breaks other uses.